src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write code here
+const ulList = [...document.querySelector('ul').children];
+const newList = document.querySelector('ul');
+
+function formatSalary(employee) {
+  const salary = employee.dataset.salary;
+  const formattedSalary = +salary.replaceAll(',', '').slice(1);
+
+  return formattedSalary;
```

Comment:
then if you pas only salary you can just write only one line of code inside function body
```suggestion
return  +salary.replaceAll(',', '').slice(1);
```